profiles/base.yaml

```diff
@@ -333,7 +333,8 @@ alchemy_room:
 use_own_ingot_type:
 deed_own_ingot: false
 crafting_training_spells:
-
+craft_max_mindstate: 31
+yarn_quantity: 100
```

Comment:
❓ Why is this a setting? i.e. when would a user want to change this from the default?

Reply:
which one? Carrying more yarn means less trips to the shop and it's light. but some people might run workorders also which trashes yarn so there is no point buying extra. the max mindstate is so I can have it not craft if i'm already at a high mindstate for some reason (having run workorders in crossing-training beforehand) I only want 12 mindstates, happy to remove it though.

Reply:
Ok